images/router/haproxy-base/Dockerfile

```diff
@@ -14,7 +14,7 @@ FROM openshift/origin-base
 RUN yum -y install haproxy && \
     mkdir -p /var/lib/containers/router/{certs,cacerts} && \
     mkdir -p /var/lib/haproxy/{conf,run,bin,log} && \
-    touch /var/lib/haproxy/conf/{{os_http_be,os_edge_http_be,os_tcp_be,os_sni_passthrough,os_reencrypt}.map,haproxy.config} && \
+    touch /var/lib/haproxy/conf/{{os_http_be,os_edge_http_be,os_tcp_be,os_sni_passthrough,os_reencrypt,os_edge_http_expose,os_edge_http_redirect}.map,haproxy.config} && \
```

Comment:
Not really but it does tell you all the maps there are. I think we wanted  to get rid of this image and so will end up having to merge into the haproxy dockerfile - fold it into  ```images/router/haproxy/Dockerfile```.